modules/tensor_mechanics/test/include/userobjects/TestSubblockIndexProvider.h

```diff
@@ -10,13 +10,14 @@
 #pragma once
 
 #include "SubblockIndexProvider.h"
+#include "ElementIntegralUserObject.h"
 
 /**
  * A class used to set the subblock index for testing generalized plane strain
  * calculations when more than one out-of-plane strain is provided on different
  * subsets of elements.
  */
-class TestSubblockIndexProvider : public SubblockIndexProvider
+class TestSubblockIndexProvider : public SubblockIndexProvider, public ElementIntegralUserObject
```

Comment:
This used to be a `GeneralUserObject` before, so why make it an `ElementIntegralUserObject`?